certificates/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Taken from https://gitlab.com/gitlab-org/build/CNG/-/blob/master/alpine-certificates/Dockerfile
+ARG FROM_IMAGE=alpine
+ARG ALPINE_VERSION=3.14
+FROM $FROM_IMAGE:$ALPINE_VERSION
+
+ARG CA_PKG_VERSION=20191127-r5
```

Comment:
Is there something newer to start with?

Reply:
@ableuler for `alpine:3.14` that is all there is:
https://pkgs.alpinelinux.org/packages?name=ca-certificates&branch=v3.14

Reply:
Hmm, ok - could you quickly / did you check if a container with root certs injected by this image actually works with letsencrypt certs after their recent change?

Reply:
ie that from inside such a container you can for example resolve renkulab.io?